discovery/kubernetes/kubernetes.go

```diff
@@ -48,8 +51,96 @@ var (
 		},
 		[]string{"role", "event"},
 	)
+	// DefaultSDConfig is the default Kubernetes SD configuration
+	DefaultSDConfig = SDConfig{}
+)
+
+// Role is role of the service in Kubernetes.
+type Role string
+
+// The valid options for Role.
+const (
+	KubernetesRoleNode     Role = "node"
+	KubernetesRolePod      Role = "pod"
+	KubernetesRoleService  Role = "service"
+	KubernetesRoleEndpoint Role = "endpoints"
+	KubernetesRoleIngress  Role = "ingress"
```

Comment:
Ah, I meant removing the `Kubernetes` prefix in the constant names as well (not just the type name).